### ISSUE TITLE
Support Electron 2-0-x?

### ISSUE THREAD

**Issue by qazbnm456** [MarshallOfSound/electron-devtools-installer#73]
Hi, since the electron 2-0-x series has just come out, will electron-devtools-installer support this series as well?

Currently, it only supports `^1.2.1` to install most dev extensions. 😞

**Comment by jeremejevs** [comment]
Don't know if this is Electron-2-specific or not, but I get errors looking like this (not together) when running `installExtension` in the main process:

```
[snip:ERROR:CONSOLE(snip)] "Extension server error: Operation failed: : has no execution context", source: chrome-devtools://devtools/bundled/inspector.js (snip)
[snip:ERROR:CONSOLE(snip)] "Extension server error: Object not found: <top>", source: chrome-devtools://devtools/bundled/inspector.js (snip)
```

Moving to the renderer has helped.

**Comment by MarshallOfSound** [comment]
Fixed in `2.2.4`

**Comment by sxcooler** [comment]
I've got the same issue here, my project is based on https://github.com/chentsulin/electron-react-boilerplate, and I replaced REDUX_DEVTOOLS with MOBX_DEVTOOLS and it worked well.
But when I change main.dev.js's mainWindow.loadUrl from 'file://xxx' to 'http://localhost:xxxx/xxx', the devtools-installer crashes.
And the console spams "...Extension server error: Object not found: <top>..."
